### PR TITLE
Add Makefile for patient generation for EXM105

### PR DESCRIPTION
**Pull request by mgramigna**
[Companion Synthea PR](https://github.com/projecttacoma/synthea/pull/6)

Add Makefile for generating EXM105 patients. I decided to seed cqf-ruler with the full measure bundle now that the connectathon repo has them. 

*NOTE*: Change the synthea branch in the Makefile to `exm105` during testing until the Synthea PR is merged into `abacus`

### CODE REVIEW

File: EXM105/Makefile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+synthea:
+	git clone --single-branch --branch abacus --depth 1 https://github.com/projecttacoma/synthea.git
+
+PATIENT_COUNT := 30
+generate-patients:
+	cd synthea && ./run_synthea -a 18-21 -p $(PATIENT_COUNT) -m stroke_exm_105
+
+CQL_FILE := connectathon/fhir3/bundles/EXM105_FHIR3-8.0.000/EXM105_FHIR3-8.0.000-files/EXM105_FHIR3-8.0.000.cql
+calculate-patients:
+	calculate-bundles -d ./synthea/output/fhir_stu3 -c $(CQL_FILE) -u http://localhost:8080/cqf-ruler-dstu3/fhir
```
Comment (lukoktonos) [inline EXM105/Makefile:33]: We should probably add instructions to do `npm install -g fhir-bundle-calculator` in the README of this repo

**Review by lukoktonos — CHANGES_REQUESTED** [review]
Makefile working using the exm105 synthea branch. 1 minor suggestion to add a line to the README then looks good.

File: EXM105/Makefile (hunk 1 above)
Comment (mgramigna) [reply EXM105/Makefile:33]: Good call. Added details to README

**Review by lukoktonos — APPROVED** [review]

**Review by okeefm — APPROVED** [review]
Getting ~1/3 to ~1/2 Numerator patients, a handful of Denominator/IPOP patients, and the rest No Population patients, which sounds good to me.

**Pull request merged**